src/bitte_plugin/src/handler/open_api_handler.rs

```diff
@@ -83,13 +85,14 @@ pub async fn open_api_specification() -> Json<Value> {
             "/api/tweet-contract-call": {
               "get": {
                 "summary": "Reward Transaction Request data/ create transaction",
-                "description": "An array of transaction objects to be signed by user to Reward Request/Reward reserve",
-                "operationId": "transaction",
+                "description": "A transaction objects to be signed by user to Reward Request/Reward reserve",
```

Comment:
**Note:** The Bitte LLM uses these `description` and `summary` properties to understand how to execute.

The `operationId` shows on Bitte - ie. what action identifier is being executed.